models/employees/employee_preferences/preferences.py

```diff
@@ -6,7 +6,7 @@
 
 @dataclass
 class Preferences:
-    days_cannot_work: list[DayOffPreference] = field(default_factory=list)
-    days_prefer_not_to_work: list[DayOffPreference] = field(default_factory=list)
+    days_cannot_work: DayOffPreference = field(default_factory=DayOffPreference)
```

Comment:
the fact i need to specify `days_cannot_work` and `shifts_cannot_work` sucks. i would like to use any ShiftsPreferences for any of these